### PR TITLE
Allow setting a default value for a parameter using the modifier

### PR DESCRIPTION
**Pull request by jotamartos**
### Description of the change

These changes allow setting a default value for a parameter in the comment itself. That way, the tool won't take into account the value of the parameter when building the README file 

### Benefits

This allows using placeholders in the registry and repository tags of the README if necessary

### Possible drawbacks

None

### Checklist

- [X] Version bumped in `package.json` and `package-lock.json` according to [semver](http://semver.org/).
- [X] New features are documented in the README.md
- [X] New features contain a new test at the `tests` folder
- [X] All tests pass running `npm test`

### CODE REVIEW

File: tests/test-values.yaml [hunk 1]
```diff
@@ -59,9 +59,9 @@ hostAliases: []
 ## Bitnami Kubewatch image version
 ## ref: https://hub.docker.com/r/bitnami/kubewatch/tags/
 ##
-## @param image.registry Kubewatch image registry
-## @param image.repository Kubewatch image name
-## @param image.tag Kubewatch image tag
+## @param image.registry [default: REGISTRY_NAME] Kubewatch image registry
+## @param image.repository [default: REPOSITORY_NAME/kubewatch] Kubewatch image name
+## @skip image.tag Kubewatch image tag
 ## @param image.pullPolicy Kubewatch image tag
```
Comment (fmulero) [inline tests/test-values.yaml:65]: Aaaargh! please accept this change I was getting crazy thinking that the `skip` tag was doing something wrong
```suggestion
## @param image.pullPolicy Kubewatch image pull policy
```

File: tests/expected-schema.json [hunk 2]
```diff
@@ -65,7 +65,7 @@
                 },
                 "tag": {
                     "type": "string",
-                    "description": "Kubewatch image tag",
+                    "description": "",
                     "default": "0.1.0-debian-10-r162"
                 },
```
Comment (fmulero) [inline tests/expected-schema.json:70]: Shouldn't it be ignored also in the schema? Also the default value stablished in the modifier should apply to the  schema also.

**Review by fmulero — COMMENTED** [review]
Thanks a lot @jotamartos.

I left some comments regarding to the schema generation, could you review them?

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bitnami/readme-generator-for-helm",
-  "version": "2.5.2",
+  "version": "2.5.3",
```
Comment (fmulero) [inline package.json:3]: This is a new feature, I think we should bump a minor.

**Review by carrodher — APPROVED** [review]

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**